src/composables/currencies.ts

```diff
@@ -53,7 +53,7 @@ const initPollingWatcher = createPollingBasedOnMountedComponents(POLLING_INTERVA
 export function useCurrencies({
   withoutPolling = false,
   store,
-  selectedProtocol = PROTOCOL_AETERNITY, // TODO - remove default value & make the protocol required
+  selectedProtocol = PROTOCOL_AETERNITY,
```

Comment:
I suggest to rename this into `protocol`. "selected" adds no context here.

Reply:
this will be removed in #2327